packages/core/src/components/IconLink/IconLink.css

```diff
@@ -0,0 +1,25 @@
+
+.iconLink {
+  color: var(--color-black);
+  font-family: var(--font-avenir);
+  font-weight: var(--fontweight-demi);
+  font-size: var(--fontsize-small-i);
+  text-transform: uppercase;
+  text-decoration: none;
+  padding-bottom: 0.5rem;
+  border-bottom: 1px solid var(--color-black);
```

Comment:
Would `rem` be better here?

Reply:
I wouldn't say so when it's only 1 pixel to be honest but I like the thinking 😉